integrations/cdi/datasource-ucp/src/main/resources/META-INF/native-image/io.helidon.integrations.cdi/helidon-integrations-cdi-datasource-ucp/reflect-config.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "oracle.ucp.jdbc.PoolDataSourceImpl",
```

Comment:
Ideally this file would be present in UCP itself, right? Just trying to understand, since this class has nothing to do with the extension.

Reply:
Whenever we need to add such a configuration to Helidon, it means it should be part of the library.
GraalVM is preparing some type of repository which would decouple the native configuration from the libraries, not sure what the state is right now

Reply:
Right, ideally it should be there i.e. with library but since its not we are adding that. Metadata for this lib is also not 
 in GraalVM repo https://github.com/oracle/graalvm-reachability-metadata as of now.